_sass/_layout.scss

```diff
@@ -6,7 +6,9 @@ body {
   padding-bottom: 70px;
   color: var(--global-text-color);
   background-color: var(--global-bg-color);
-
+  unicode-bidi: bidi-override !important;
+  direction: unset !important;
```

Comment:
sorry, I'm not as familiar with multi-directional language support. what does this do?

Reply:
TBH, I don't know much rather than `bidi` handles bi-directional scripting in the correct way. Often right-to-left scripts are not equal to a right-aligned version of a left-to-right script. Text becomes more shattered if there are numbers or special characters involved as well. Yet, with smart conditioning, one can prevent the misplacement of phrases. I guess, what bidi does is take care of all those cases.

Why these particular key-values are necessary to fix this, I don't know. I had [this](https://stackoverflow.com/q/19997153/7106957) problem and (almost) followed the accepted answer. In my case, it worked.